backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc

```diff
@@ -42,6 +43,7 @@ include::modules/restore-replace-stopped-etcd-member.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 * xref:../../machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc#cpmso-ts-etcd-degraded_cpmso-troubleshooting[Recovering a degraded etcd Operator]
+* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/expanding-the-cluster#installing-primary-control-plane-node-unhealthy-cluster_expanding-the-cluster[Replacing a control plane node in an unhealthy cluster (for Assisted Installer)]
```

Comment:
I always point this out:

This technically links to "Installing a primary control plane node on an unhealthy cluster". I'm guessing that this is intentional? I always think it's odd when we change the friendly reference text from the subheading text. Could it potentially confuse someone?

Reply:
@stevsmit, thanks so much for the review! I can change the titles back, though you're right that the change was intentional. @tshwartz has been updating links and file names for some of this content, and we agreed to the change.